src/modules/search/components/SearchBar/SearchBar.js

```diff
@@ -67,8 +67,8 @@ const SearchBar = ({ className }) => {
   const onSelectBlock = (value) => onSelectedRow('block', value);
 
   const onKeyPress = () => {
-    if (addresses.length) {
-      onSelectAccount(addresses[rowItemIndex].address);
+    if (Object.keys(addresses).length) {
+      onSelectAccount(addresses.address);
     }
```

Comment:
instead of this I think you can assert by this

```suggestion
    if (addresses.address) onSelectAccount(addresses.address)
```

Reply:
This was done for consistency with the other conditions in the function